soc/arm/nuvoton_npcx/common/soc_ncl.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2024 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _NUVOTON_NPCX_SOC_NCL_H_
+#define _NUVOTON_NPCX_SOC_NCL_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* The status code returns from Nuvoton Cryptographic Library ROM APIs */
+enum ncl_status {
+	NCL_STATUS_OK = 0xA5A5,
+	NCL_STATUS_FAIL = 0x5A5A,
+	NCL_STATUS_INVALID_PARAM = 0x02,
+	NCL_STATUS_PARAM_NOT_SUPPORTED,
```

Comment:
I think it may be a good idea to define all value explicitly since some are overridden

Reply:
Done.